ext/mysqli/mysqli.stub.php

```diff
@@ -0,0 +1,540 @@
+<?php
+
+class mysqli_driver
+{
+}
+
+class mysqli
+{
+    /** @return mysqli|null|false */
+    public function __construct(
+        ?string $host = null,
+        ?string $user = null,
+        ?string $password = null,
+        ?string $database = null,
+        ?int $port = null,
+        ?string $socket = null
+    );
+
+    /** @return bool|null */
+    public function autocommit(bool $mode);
```

Comment:
@nikic Lots of mysqli functions invoke `MYSQLI_FETCH_RESOURCE_CONN` or `MYSQLI_FETCH_RESOURCE`, which can return `NULL` or `FALSE` (and emit a warning) in case of an issue. Could we promote these warnings to exceptions instead?

Reply:
I think so.

Reply:
Thanks! I'll try to provide a PR then. @tvlooy If we can get rid of the warnings, then unfortunately all the return values should be reevaluated :( But at least, we can also eliminate these return values from the stubs.